src/shogun2-webapp-archetype/src/main/resources/archetype-resources/src/main/resources/META-INF/spring/__artifactId__-context-initialize-beans.xml

```diff
@@ -425,14 +425,12 @@
 
     <bean id="osmWmsLayer" class="de.terrestris.shogun2.model.layer.Layer">
         <property name="name" value="OSM-WMS" />
-        <property name="type" value="Tile" />
```

Comment:
This removement raises an issue in SHOGun2-client project while [creating layer sources](https://github.com/terrestris/shogun2-client/blob/master/app/view/component/MapController.js#L133). I know SHOGun2-client is a separate project but somehow related.